src/Microsoft.Health.Fhir.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -55,6 +55,9 @@ internal class KnownRoutes
         public const string MemberMatch = "Patient/$member-match";
         public const string MemberMatchOperationDefinition = OperationDefinition + "/" + OperationsConstants.MemberMatch;
 
+        public const string Everything = "$everything";
+        public const string PatientEverythingById = "Patient/" + IdRouteSegment + "/" + Everything;
```

Comment:
>PatientEverythingById [](http://example.com/codeflow?start=28&length=21)

Just to double check.
[https://www.hl7.org/fhir/operation-patient-everything.html](https://www.hl7.org/fhir/operation-patient-everything.html)

define two endpoints:

URL: [base]/Patient/$everything

URL: [base]/Patient/[id]/$everything

We don't add first endpoint yet, because it's low priority?

Reply:
This is because there is a discussion about Patient level export. Patient level export accomplishes a similar functionality as the first endpoint, which pulls all patient data into a storage account. Now we will direct customers to continue to use patient export if they need all patients. In the future, we may want to update the Patient level export to use the same logic as $patient-everything. The details could be found in the PM spec.